docs/index.html

```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Toolkit for Bare Metal Service/</title>
```

Comment:
Is that trailing slash on "Service/" supposed to be there?

Reply:
Nope, fixed.